run_linux.go

```diff
@@ -1836,7 +1855,9 @@ func (b *Builder) runSetupVolumeMounts(mountLabel string, volumeMounts []string,
 				return specs.Mount{}, errors.Wrapf(err, "failed to create TempDir in the %s directory", containerDir)
 			}
 
-			overlayMount, err := overlay.Mount(contentDir, host, container, rootUID, rootGID, b.store.GraphOptions())
+			overlayOpts := overlay.Options{RootUID: rootUID, RootGID: rootGID, UpperDirOptionFragment: upperDir, WorkDirOptionFragment: workDir, GraphOpts: b.store.GraphOptions()}
```

Comment:
Purely aesthetic: Maybe use one value per line here, to make it more readable?

(My personal preference would be to embed the value that into the `MountWithOptions` call, but that’s entirely up to the maintainers of this repo.)

Reply:
fixed